src/platform/plugins/shared/discover/public/application/main/components/layout/use_discover_histogram.ts

```diff
@@ -267,18 +268,6 @@ export const useDiscoverHistogram = ({
    * Data fetching
    */
 
-  const skipRefetch = useRef<boolean>();
```

Comment:
it's great that we can remove this logic here! 🥳

Reply:
I agree! I've always disliked this workaround and have been wanting to remove it for a while, and now I finally had a good reason 😁